Shared/Samples/Generate offline map/GenerateOfflineMapView.swift

```diff
@@ -30,9 +33,9 @@ struct GenerateOfflineMapView: View {
             MapViewReader { mapView in
                 MapView(map: model.offlineMap ?? model.onlineMap)
                     .interactionModes(isGeneratingOfflineMap ? [] : [.pan, .zoom])
-                    .errorAlert(presentingError: $model.error)
+                    .errorAlert(presentingError: $error)
                     .task {
-                        await model.initializeOfflineMapTask()
+                        try? await model.initializeOfflineMapTask()
```

Comment:
```suggestion
                        do {
                            try await model.initializeOfflineMapTask()
                        } catch {
                            self.error = error
                        }
```
The result of `try` is ignored here. If the task fails to initialize should the error be presented to the user?

Reply:
Yeah, I think it's worth letting the user know if the load request times out.